src/modelframe.jl

```diff
@@ -79,6 +117,15 @@ function evalcontrasts(df::AbstractDataFrame, contrasts::Dict = Dict())
     return evaledContrasts
 end
 
+## Default NULL handler.  Others can be added as keyword arguments
```

Comment:
oops, yes, I'd made a commit on master that I didn't push.  I'll rebase this all appropriately before merging.